app/controllers/payments_controller.rb

```diff
@@ -0,0 +1,36 @@
+class PaymentsController < ApplicationController
+  # before_action :authenticate_user!
```

Comment:
- [ ] Please kindly delete the commented line of code to make your codebase clean and professional 🙃